packages/frontend/src/layouts/MainView.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import HomePage from '../pages/HomePage';
+import Movies from '../pages/Movies';
+import Series from '../pages/Series';
+
+const MainView: React.FC = () => {
+    return (
+        <>
+            <Route path="/" exact component={HomePage} />
```

Comment:
Mozesz dodać Switcha to ułatwi zadanie https://reacttraining.com/react-router/web/guides/quick-start

Reply:
Nie tyle możesz co powinnaś. Dlatego że bez switch może to spowodować że wszystkie komponenty których url pasuje do wzorca się wydrukują a raczej chcemy by wydrukował sie pierwszy który pasuje do wzorca.

Reply:
Zmienione na Switch ;)